docs/receipts.md

```diff
@@ -0,0 +1,42 @@
+# Document Generation in Indico
+
+Indico includes a module for generating documents, such as receipts and certificates, from templates that can be defined in the event or category. This feature is especially useful for conference organizers and administrators who need to provide attendees with customized documents quickly and efficiently.
```

Comment:
We already mention this under `## Creating a template` so maybe just

```diff
- from templates that can be defined in the event or category
+ from predefined templates
```

Reply:
Since this is an introduction of the feature I don't think it's that bad to repeat that up here... I think we can assume most people only read the beginning anyway